code/game/gamemodes/changeling/powers/synaptic_enhancement.dm

```diff
@@ -4,25 +4,23 @@
 	helptext = "The delay between attacks and actions will be halved. This will halt our chemical regeneration."
 	evopoints_cost = 8 //This is a VERY powerful lil piece of shit, especially if you use it in conjunction with a high-damage weapon.
 	req_dna = 6 //Tier 3
-	chemical_cost = 0 //Chemical cost is hardcoded
+	chemical_cost = 30
 	var/active = FALSE
 
 /obj/effect/proc_holder/changeling/synaptic_enhancement/sting_action(mob/user)
 	var/datum/changeling/changeling=user.mind.changeling
 	if(!active)
-		if(chemical_cost <= 30) //Hardcoded chem cost
-			user << "<span class='warning'>We require at least 30 units to do that!</span>"
-			return
 		active = 1
-		changeling.chem_storage -= 30
 		user.next_move_modifier = 0.5
+		chemical_cost = 0
```

Comment:
what no you don't have to set it later on, remove it from here and  on the line 23